frontend/src/components/shared/TooltipIcon/styled-components.ts

```diff
@@ -16,17 +16,38 @@
 
 import styled from "@emotion/styled"
 
-export const StyledTooltipIconWrapper = styled.div(({ theme }) => ({
-  display: "flex",
-  alignItems: "center",
+interface StyledTooltipIconWrapperProps {
+  isLatex?: boolean
+}
+
+export const StyledTooltipIconWrapper =
+  styled.div<StyledTooltipIconWrapperProps>(({ isLatex, theme }) => ({
+    display: "flex",
+    alignItems: "center",
+    marginTop: isLatex ? "12px" : "0",
```

Comment:
Could we potentially use some theme item as an alternative to hard-coded `12px` here?